docs/releaseNotes.yml

```diff
@@ -35,13 +35,29 @@ items:
     prevVersion: 2.1.0
     date: 'TBD'
     notes:
+
       - title: Docker BuildKit always used for builds
         type: change
         body: >-
           Docker BuildKit is enabled for all Emissary builds. Additionally, the Go
           build cache is fully enabled when building images, speeding up repeated builds.
         docs: https://github.com/moby/buildkit/blob/master/frontend/dockerfile/docs/syntax.md
 
+      - title: Fix overriding global settings for adding or removing headers
+        type: bugfix
+        body: >-
+          If the <code>ambassador</code> <code>Module</code> sets a global default for
+          <code>add_request_headers</code>, <code>add_response_headers</code>,
+          <code>remove_request_headers</code>, or <code>remove_response_headers</code>, it is often
+          desirable to be able to turn off that setting locally for a specific <code>Mapping</code>.
+          For several releases this has not been possible for <code>Mappings</code> that are native
+          Kubernetes resources (as opposed to annotations), as an emtpy value ("mask the global
+          default") was erroneously considered to be equivalent to unset ("inherit the global
+          default").  This is now fixed.
+        docs: topics/using/defaults/
+
+      # next changelog item here
```

Comment:
Why is this left in here?

Reply:
To reduce conflicts for other outsdanding PRs.